### PR TITLE
Add imi Caching test

### PR DESCRIPTION
**Pull request by Yurunsoft**

### CODE REVIEW

File: frameworks/PHP/imi/imi-raw.dockerfile [hunk 1]
```diff
@@ -5,18 +5,25 @@ RUN pecl install swoole > /dev/null && \
...
 RUN docker-php-ext-install pdo_mysql > /dev/null
 
+RUN pecl install redis > /dev/null && \
+    docker-php-ext-enable redis
+
 RUN apt -yqq update > /dev/null && \
     apt -yqq install git unzip > /dev/null
 
-WORKDIR /imi
+RUN apt -yqq install redis-server > /dev/null
```
Comment (nbrady-techempower) [inline frameworks/PHP/imi/imi-raw.dockerfile:14]: Is there a reason you're installing redis for tests that aren't doing cached queries?

Comment (Yurunsoft) [reply frameworks/PHP/imi/imi-raw.dockerfile]: I have removed non-essential redis

**Comment by nbrady-techempower** [comment]
@Yurunsoft Can you pull in the latest to get the mysql update that's blocking this?

**Comment by Yurunsoft** [comment]
@nbrady-techempower Thank you very much, it has been fixed

**Pull request merged**